AmplifyPlugins/Predictions/CoreMLPredictionsPlugin/Dependency/CoreMLSpeechAdapter.swift

```
@@ -12,12 +12,24 @@ class CoreMLSpeechAdapter: CoreMLSpeechBehavior {
     func getTranscription(_ audioData: URL) async throws -> SFSpeechRecognitionResult {
         let request = SFSpeechURLRecognitionRequest(url: audioData)
         request.requiresOnDeviceRecognition = true
-        let recognizer = SFSpeechRecognizer()
+        guard let recognizer = SFSpeechRecognizer() else {
+            throw PredictionsError.client(
+                .init(
+                    description: "CoreML Service is not configured",
+                    recoverySuggestion: "Ensure that dictation is enabled on your device."
+                )
```

In the spirit of failing fast and early.